### ISSUE TITLE
Gruntfile: fix widget configuration

### ISSUE THREAD

**Issue by x1B** [LaxarJS/grunt-init-laxar-application#28]
The changes for 1.0.0-beta.0 broke the auto-configuration for widgets.

**Comment by x1B** [comment]
Also, the require-configuration is missing a path entry for tests:

```js
      jquery: 'jquery/dist/jquery',
```

**Comment by x1B** [comment]
Also, add the shim for `json-patch`:

```js
      // LaxarJS Patterns
      'json-patch': {
         exports: 'jsonpatch'
      }
```

**Comment by x1B** [comment]
Fixed on `master` (`v1.0.0-beta.1`).
